scripts/db_migrator.py

```diff
@@ -422,6 +422,23 @@ def migrate_pfcwd_sw_enable_table(self):
                 v['pfcwd_sw_enable'] = v['pfc_enable']
                 self.configDB.set_entry('PORT_QOS_MAP', k, v)
 
+    def migrate_port_qos_map_global(self):
+        """
+        Generate dscp_to_tc_map for switch.
+        """
+        asics_require_global_dscp_to_tc_map = ["broadcom"]
+        if self.asic_type not in asics_require_global_dscp_to_tc_map:
+            return True
```

Comment:
"return" should do since we are not capturing the return value of this function

Reply:
Fixed. Thanks